src/components/Link.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link as ChakraLink, useTheme } from '@chakra-ui/core';
+import { Link as GatsbyLink } from 'gatsby';
+
+function Link(props) {
+  const theme = useTheme();
+  const isCallToAction = props.isCallToAction ? true : false;
```

Comment:
I believe the preferred way to do this kind of check is
```suggestion
  const isCallToAction = (!!props.isCallToAction);
```

This has to do with _truthy_ and _falsy_ values in javascript, and will ensure that only boolean values work here.  As it's written right now, this would render as a CTA link, which is counterintuitive:

```jsx
<Link isCallToAction={5} />
```

Reply:
Thanks that's a very good point! 

My thinking was that the component would be used like this so I was just checking whether the prop was present:

```jsx
<Link isCallToAction />
```
Maybe not good practice.

I'm now looking at the styles for links in the footer, maybe makes more sense to have a type prop and do something like this:

```jsx
<Link /> // no prop for default link
<Link type="callToAction" />
<Link type="footer" />
```
Do you think this would make more sense? Rather than

```jsx
<Link isCallToAction={true} />
<Link isFooterLink={true} />
```

Reply:
Whenever I've written `<comp prop={true}>` in my projects it always gets auto corrected to `<comp prop>` due to [this linting rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md)

Reply:
> Whenever I've written `<comp prop={true}>` in my projects it always gets auto corrected to `<comp prop>` due to [this linting rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md)

I think I've had this before myself actually...

It's late where I am right now so I'm going to bed. I'll work it out in the morning :)

Reply:
@racedale and @subject026,
In the `<Button>`  component we are currently using a `variant` prop that takes in one of the three:
- cta
- primary
- secondary

So I think for this `variant` would be the correct way to go with the following options:
- cta
- footer

And the styles used to theme these links should probably be contained in the `src\gatsby-plugin-chakra-ui\theme.js` file.

Reply:
@ChrisBrownie55 @mbifulco @racedale Thanks for the comments! 

Made some changes and logic is now more consistent with the Button component. 3 variants now to cover standard, cta and footer links.